ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- The title above should be a short description of the issue. -->
+
+#### What is the problem?
+
+#### Example of problem
```

Comment:
`The code to reproduce the problem`

Reply:
@smichr Is it ok to write this?

Reply:
Yes - it tell more specifically what is expected: code.